src/pages/Checkout/index.js

```diff
@@ -21,10 +21,10 @@ const Checkout = () => {
             <div className="checkout-content">
               <Title level={2}>CHECKOUT DETAILS</Title>
               <Row style={{ gap: '2rem' }}>
-                <Col span={14}>
+                <Col xxl={14} xl={14} lg={14} md={24} sm={24} xs={24}>
```

Comment:
You should add a breakpoint prop when the num of column change. In this case, you just need two props
lg={14}  xs={24}